File: Array/1672-Richest-Customer-Wealth/1672-Richest-Customer-Wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    let ans = 0;

    accounts.map(amounts =>{
        let sum = amounts.reduce((acc, curr) => acc+curr);

        ans = (ans < sum) ? sum : ans;
    })
    return ans;
};

File: Array/1672-Richest-Customer-Wealth/1672-Richest-Customer-Wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        
        int ans = 0;

        for(auto amounts : accounts) {
            int sum = 0;
            for(auto value : amounts) {
                sum+=value;
            }
            ans = max(sum, ans);
        }
        return ans;
    }
};
